docker/dockerfile-build-ubuntu-rock

```diff
@@ -0,0 +1,48 @@
+# This Dockerfile provides a starting point for a ROCm installation of hipCaffe.
+
+# Parameters related to building hip
+ARG base_image
+
+FROM ${base_image}
+LABEL maintainer="kent.knox@amd"

```

Comment:
kent is no longer the maintainer

Reply:
he's maintaining rocBLAS too apparently! Should be changed (though I don't know who the real maintainer is...maybe Saad?), but I don't want this to trigger another CI build at the moment.